man/rgbgfx.1

```diff
@@ -125,11 +125,14 @@ begins with a hash character
 .Ql # ,
 it is treated as an inline palette specification.
 It should contain a comma-separated list of hexadecimal colors, each beginning with a hash.
-Colors in are accepted either as
+Colors are accepted either as
 .Ql #rgb
 or
 .Ql #rrggbb
 format.
+To leave a gap in the palette,
+.Ql #none
+can be used instead of one or more colors.
```

Comment:
```suggestion
To leave one or more gaps in the palette,
.Ql #none
can be used instead of any color.
```

I *think* this expresses the intent more clearly? `#none` can be used zero or more times, but each time substitutes for exactly one colour...